src/test/java/com/linagora/elasticsearch/metrics/ElasticsearchReporterTest.java

```diff
@@ -139,6 +145,7 @@ public void testThatTemplateIsNotOverWritten() throws Exception {
 
     @Test
     public void testThatTimeBasedIndicesCanBeDisabled() throws Exception {
+        elasticsearchReporter.close();
```

Comment:
why do you close it at the beginning? cleanup is not sufficient?

Reply:
Not when we replace the instance by a new one, the following line. Like it is in this test.

Reply:
given that half of the test redefine the elasticsearchReporter, I would find it cleaner if every test define the elasticsearchReporter